### ISSUE TITLE
[BUG] Deserializing a Base with an Organization property throws a NullReferencException

### ISSUE THREAD

**Issue by tshowers-bt** [elastic/ecs-dotnet#178]
**ECS integration/library project(s) (e.g. Elastic.CommonSchema.Serilog)**: Elastic.CommonSchema

**ECS schema version (e.g. 1.4.0)**: 

**ECS .NET assembly version (e.g. 1.4.2)**: 1.5.3

**.NET framework / OS**: Windows

**Description of the problem, including expected versus actual behavior**:

Deserializing a serialized `Elastic.CommonSchema.Base` that has a non-null value for `Organization` throws a `NullReferenceException`.  I would expect to return an instance of `Elastic.CommonSchema.Base` with the same values as the the instance that was originally serialized.

edit:
The same thing happens for non-null values for `Agent` and `Client` but not for `Labels`.

Stacktrace:
```
   at System.Text.Json.JsonSerializer.LookupProperty(Object obj, ReadOnlySpan`1 unescapedPropertyName, ReadStack& state, JsonSerializerOptions options, Boolean& useExtensionProperty, Boolean createExtensionProperty)
   at System.Text.Json.Serialization.Converters.ObjectDefaultConverter`1.OnTryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonResumableConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at Elastic.CommonSchema.Serialization.EcsJsonConverterBase`1.ReadProp[TValue](Utf8JsonReader& reader, String key)
   at Elastic.CommonSchema.Serialization.EcsJsonConverterBase`1.ReadProp[TValue](Utf8JsonReader& reader, String key, T b, Action`2 set)
   at Elastic.CommonSchema.Serialization.BaseJsonConverter`1.ReadProperties(Utf8JsonReader& reader, TBase ecsEvent, Nullable`1& timestamp, String& loglevel)
   at Elastic.CommonSchema.Serialization.BaseJsonConverter`1.Read(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options)
   at System.Text.Json.Serialization.JsonConverter`1.TryRead(Utf8JsonReader& reader, Type typeToConvert, JsonSerializerOptions options, ReadStack& state, T& value)
   at System.Text.Json.Serialization.JsonConverter`1.ReadCore(Utf8JsonReader& reader, JsonSerializerOptions options, ReadStack& state)
   at System.Text.Json.JsonSerializer.ReadFromSpan[TValue](ReadOnlySpan`1 utf8Json, JsonTypeInfo jsonTypeInfo, Nullable`1 actualByteCount)
   at System.Text.Json.JsonSerializer.ReadFromSpan[TValue](ReadOnlySpan`1 json, JsonTypeInfo jsonTypeInfo)
   at System.Text.Json.JsonSerializer.Deserialize[TValue](String json, JsonSerializerOptions options)
   at Elastic.CommonSchema.Serialization.EcsSerializerFactory`1.Deserialize(String json)
   at Elastic.CommonSchema.Base.Deserialize(String json)
   at Program.<Main>$(String[] args) in C:\Users\tshowers\src\EcsDeserBug\EcsDeserBug\Program.cs:line 14
```

**Steps to reproduce**:

```cs
using Elastic.CommonSchema;

var x = new Base
{
    Timestamp = DateTime.UtcNow,
    Organization = new Organization
    {
        Id = Guid.NewGuid().ToString(),
    }
};

var ser = x.Serialize();

var y = Base.Deserialize(ser);
```

**Comment by Insomniak47** [comment]
FYI I bisected the repo and found that it *is* fixed on main. The commit that fixes it is: 91771540280c473580a2b5161d977620ea7a7e3f

It seems the dotnet 5 update fixed it though it likely needs a fix in the 1.5.x branch

**Comment by Mpdreamz** [comment]
I added an explicit unit test for this but I can not reproduce this on `main`

https://github.com/elastic/ecs-dotnet/pull/253

Thank you for reporting this @tshowers-bt, a new version should go up on NuGet soon.
